lib/core/util.js

```diff
@@ -1,276 +1,294 @@
-'use strict'
+"use strict";
```

Comment:
I'm not sure how this file passes linting but it's not ok. Can you please restore it to its previous state?